bundle-types/action.yml

```diff
@@ -59,7 +59,7 @@ runs:
         PACKAGE_JSON_PATH: ${{ inputs.package-json-path }}
 
     - id: get-secrets
-      uses: grafana/shared-workflows/actions/get-vault-secrets@28361cdb22223e5f1e34358c86c20908e7248760 # get-vault-secrets-v1.1.0
+      uses: grafana/shared-workflows/actions/get-vault-secrets@5d7e361bc7e0a183cde8afe9899fb7b596d2659b # get-vault-secrets-v1.2.0
```

Comment:
what I mean is that maybe we want to keep getting updates automatically? (maybe we dont)

Reply:
I'm not sure I'm following? Are you suggesting using the tag and adding a [zizmor config](https://woodruffw.github.io/zizmor/audits/#rulesunpinned-usesconfigpolicies) to mark the grafana/shared-workflows as safe? I'm not sure I like the idea of pointing it at main and zizmor ignore it.

Reply:
yep, doing `@main #zizmor-ignore-whatever`. I have seen that in other references to Grafana actions so it's not a first but sure, we can leave it pinned